packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/funman/config/FunmanConfig.java

```diff
@@ -0,0 +1,32 @@
+package software.uncharted.terarium.hmiserver.models.funman.config;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+
+@Data
+@Accessors(chain = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
I wont put a comment on each, but, technically this is unneeded as we have `spring.jackson.default-property-inclusion=NON_NULL` in our application.properties

Reply:
yeah I was curious if we needed this.  I'll remove it if we don't.  as always, appreciate the explanation.